provider/provider.go

```diff
@@ -152,10 +152,12 @@ func providerConfigure(d *schema.ResourceData, terraformVersion string) (interfa
 		},
 	}
 	if v := os.Getenv("SPINNAKER_CERT"); v != "" {
+		clientConfig.Auth.Enabled = true
 		clientConfig.Auth.CertPath = v
 		clientConfig.Auth.KeyPath = os.Getenv("SPINNAKER_KEY")
 	}
 	if v := os.Getenv("SPINNAKER_CERT_CONTENT"); v != "" {
```

Comment:
This is really weird ,, could we normalize the environment variables??

Reply:
What do you mean? The original code already had `SPINNAKER_CERT` to map the certificate path. We added this `SPINNAKER_CERT_CONTENT` to assign the certificate base64 content making it easier to setup. We are following the pattern for kubectl where the certificates base 64 use the `_CONTENT` suffix.